af/vf-command: add ability to target a specific lavfi filter (#11182)
mpv-player/mpv: DOCS/man/input.rst

    The ``flags`` argument is like the first argument to ``screenshot`` and
    supports ``subtitles``, ``video``, ``window``.

``vf-command <label> <command> <argument> [<target>]``
    Send a command to the filter. Note that currently, this only works with
    the ``lavfi`` filter. Refer to the libavfilter documentation for the list
    of supported commands for each filter.

>     ``<label>`` is a mpv filter label, use ``all`` to send it to all mpv
>     filters at once.

Reviewer: ```suggestion
    ``<label>`` is a mpv filter label, use ``all`` to send it to all filters
    at once.
```

    ``<command>`` and ``<argument>`` are filter-specific strings.

>     ``<target>`` is a filter or filter instance name, if not specified,
>     default to ``all``. Note that the target is an additional specifier
>     for filters that support those, such as complex lavfi filter chains.

Reviewer: ```suggestion
    ``<target>`` is a filter or filter instance name and defaults to ``all``.
    Note that the target is an additional specifier for filters that
    support them, such as complex ``lavfi`` filter chains.
```
